include/picongpu/particles/param.hpp

```diff
@@ -1,5 +1,4 @@
-/* Copyright 2013-2023 Axel Huebl, Heiko Burau, Rene Widera, Felix Schmitt,
- *                     Marco Garten, Finn-Ole Carstens
+/* Copyright 2013-2023 Axel Huebl, Rene Widera, Sergei Bastrakov, Klaus Steiniger
```

Comment:
authors disappeared from the license header

Reply:
github is mixing removed and newly created files.